src/components/CaptureDetailDialog.js

```diff
@@ -188,7 +188,7 @@ function CaptureDetailDialog(props) {
             <Grid item>
               <Box m={4}>
                 <Typography color="primary" variant="h6">
-                  Capture <LinkToWebmap value={capture} type="tree" />
+                  Capture <LinkToWebmap value={capture} type="trees" />
```

Comment:
Do you think we could define a shared `const` object with the two possible values for `type`?
That would mean we only need to change the string literal values in one place if the web map path changes, and minimised the risk of a bug through a typo.